WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reactive.Concurrency;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Notifications;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionProcessing;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Fluent.Helpers
+{
+	public static class NotificationHelpers
+	{
+		private const int DefaultNotificationTimeout = 7;
+		private static WindowNotificationManager? NotificationManager;
+
+		public static void SetNotificationManager(Window host)
+		{
+			var notificationManager = new WindowNotificationManager(host)
+			{
+				Position = NotificationPosition.BottomRight,
+				MaxItems = 4,
+				Margin = new Thickness(0, 0, 15, 40)
+			};
+
+			NotificationManager = notificationManager;
+		}
+
+		public static void Show(string title, string message)
+		{
+			if (NotificationManager is { } nm)
+			{
+				RxApp.MainThreadScheduler.Schedule(() => nm.Show(new Notification(title, message, NotificationType.Information, TimeSpan.FromSeconds(DefaultNotificationTimeout))));
+			}
+		}
+
+		public static void Show(ProcessedResult result)
+		{
+			if (TryGetNotificationInputs(result, out var title, out var message))
+			{
+				Show(title, message);
+			}
+		}
+
+		private static bool TryGetNotificationInputs(ProcessedResult result, [NotNullWhen(true)] out string? title, [NotNullWhen(true)] out string? message)
+		{
+			title = null;
+			message = null;
+
+			try
+			{
+				bool isSpent = result.NewlySpentCoins.Any();
+				bool isReceived = result.NewlyReceivedCoins.Any();
+				bool isConfirmedReceive = result.NewlyConfirmedReceivedCoins.Any();
+				bool isConfirmedSpent = result.NewlyConfirmedReceivedCoins.Any();
+				Money miningFee = result.Transaction.Transaction.GetFee(result.SpentCoins.Select(x => x.Coin).ToArray());
+
+				if (isReceived || isSpent)
+				{
+					Money receivedSum = result.NewlyReceivedCoins.Sum(x => x.Amount);
+					Money spentSum = result.NewlySpentCoins.Sum(x => x.Amount);
+					Money incoming = receivedSum - spentSum;
+					Money receiveSpentDiff = incoming.Abs();
+					string amountString = receiveSpentDiff.ToFormattedString();
+					message = $"{amountString} BTC";
+
+					if (result.Transaction.Transaction.IsCoinBase)
+					{
+						title = "Mined";
```

Comment:
Is it the right word? I think that transactions are not "mined", blocks are.

Reply:
```suggestion
						title = "Coinbase reward";
```

maybe this, but I actually think `mined` is accurate and understandable in this situation.